### DISCUSSION TITLE
hello world

### DISCUSSION THREAD

**Discussion by turt2live** [turt2live/test#17] [Ideas]
I HAVE AN IDEA

**Comment by turt2live** [comment]
this is an answer?

**Reply by tulir** [reply]
no

**Reply by deepbluev7** [reply]
no

**Reply by deepbluev7** [reply]
tulir stole my reply, rude!

**Comment by turt2live** [comment]
how do I make an answer

**Comment by jaywink** [comment]
woop

**Comment by deepbluev7** [comment]
As a new thread

**Comment by tulir** [comment]
this is not the answer

**Reply by deepbluev7** [reply]
this is (not) an answer

**Comment by Half-Shot** [comment]
wtf is this

**Reply by turt2live** [reply]
this is the default thread using the template it gave me

**Reply by turt2live** [reply]
oh no, that's the other thread

**Reply by turt2live** [reply]
this is me spamming your inbox

**Reply by Half-Shot** [reply]
thank you

**Reply by Half-Shot** [reply]
i really hate how Enter does not send a reply. this feels too much like chat
